### PR TITLE
Fix sqlalchemy default values for insert and update queries

### PR DESCRIPTION
**Pull request by ckkz-it**
This is what actually `aiopg` does https://github.com/aio-libs/aiopg/blob/master/aiopg/sa/engine.py#L17#L30

Fixes https://github.com/encode/databases/issues/72
I think this solution is less tricky than https://github.com/encode/databases/pull/206/

### CODE REVIEW

File: tests/test_databases.py [hunk 1]
```diff
@@ -917,6 +1009,7 @@ async def run_database_queries():
         async with database:
 
             async def db_lookup():
+
```
Comment (astret-soft) [inline tests/test_databases.py:1012]: left \n

File: databases/backends/postgres.py [hunk 2]
```diff
@@ -10,11 +10,20 @@
...
+from databases.backends.common import ConstructDefaultParamsMixin
 from databases.core import LOG_EXTRA, DatabaseURL
 from databases.interfaces import ConnectionBackend, DatabaseBackend, TransactionBackend
 
 logger = logging.getLogger("databases")
 
+_result_processors = {}  # type: dict
+
+
+class APGCompiler_psycopg2(
```
Comment (acnebs) [inline databases/backends/postgres.py:22]: Seems strange to call this `APGCompiler_psycopg2`, given that the db backend used for postgres in databases is `asyncpg`, not `psycopg2`.

Comment (ckkz-it) [reply databases/backends/postgres.py:22]: this is how aiopg calls it
https://github.com/encode/databases/blob/master/databases/backends/aiopg.py#L32#L35

Comment (acnebs) [reply databases/backends/postgres.py:22]: Right but that's because aiopg is just a wrapper around psycopg2, asyncpg is not.

**Pull request closed without merging**